nightfall-deployer/contracts/Verifier.sol

```diff
@@ -59,31 +58,31 @@ library Verifier {
 
   function verificationCalculation(uint256[] memory _proof, uint256 _publicInputsHash, uint256[] memory _vk) public returns (uint) {
 
-      Proof_GM17 memory proof;
+      Proof_G16 memory proof;
       Pairing.G1Point memory vk_dot_inputs;
-      Verification_Key_GM17 memory vk;
+      Verification_Key_G16 memory vk;
 
       vk_dot_inputs = Pairing.G1Point(0, 0); //initialise
 
       proof.A = Pairing.G1Point(_proof[0], _proof[1]);
       proof.B = Pairing.G2Point([_proof[2], _proof[3]], [_proof[4], _proof[5]]);
       proof.C = Pairing.G1Point(_proof[6], _proof[7]);
 
-      vk.H = Pairing.G2Point([_vk[0],_vk[1]],[_vk[2],_vk[3]]);
-      vk.Galpha = Pairing.G1Point(_vk[4],_vk[5]);
-      vk.Hbeta = Pairing.G2Point([_vk[6],_vk[7]],[_vk[8],_vk[9]]);
-      vk.Ggamma = Pairing.G1Point(_vk[10],_vk[11]);
-      vk.Hgamma = Pairing.G2Point([_vk[12],_vk[13]],[_vk[14],_vk[15]]);
+      vk.alpha = Pairing.G1Point(_vk[4],_vk[5]);
```

Comment:
Wondering if there is any reason we don't start from 0 here? Seems like we dont use elts 0 to 3 of the vk